src/cf-tables.less

```diff
@@ -0,0 +1,484 @@
+    - name: .sortable__start-up, .sortable__start-down (modifiers)
+      codenotes:
+        - |
+          <table class="table__sortable">
+            ...
+              <th class="sortable sortable__start-up">Column Name</th>
+            ...
+          </table>
+      notes:
+        - "To sort the table on page load, use the '.sortable__start-up' and '.sortable__start-down'
+          classes."
+  tags:
+    - cf-tables
+*/
+
+// Color variables
+@table__head-bg:              #f8f8f8;
+@table__cell-bg:              #ffffff;
+@table__cell-bg_alt:          #f8f8f8;
+@table__scrolling-border:     #e3e4e5;
+@table__border:               #babbbd;
+
+// .table
+
+table {
+    .table__right-align {
+        text-align: right;
+    }
+}
+
+.table-wrapper__scrolling {
+    box-sizing: border-box;
+    overflow-x: auto;
+    overflow-y: hidden;
+    width: 100%;
+    table {
+        border: 1px solid @table__scrolling-border;
+        .table__striped();
+    }
+}
+
+.table__striped {
+    tbody > tr:nth-child(even) {
+        & > th,
+        & > td {
+            background: @table__cell-bg_alt;
+        }
+    }
+}
+
+table.table__sortable {
+    .sortable:after {
+        display: inline-block;
+        content: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA4AAAAJCAYAAAACTR1pAAAAXUlEQVR4AWPABZw9/C4A8Qlscvg0LQXi/1A8k1hNjXBNCFxNSFMcXDEmjsOlyQakgAB2RNckD8S/idD4D4jVYZq4gfgJXJIwfgPEIiCNt0ECJOKXII13gPgTifgaAHk0kQqUakHYAAAAAElFTkSuQmCC);
+        position: relative;
+        top: -1px;
+        visibility: hidden;
+    }
+
+    .sortable:hover:after,
+    .sortable.sorted_up:after,
+    .sortable.sorted_down:after {
+        visibility: visible;
+    }
+
+    .sortable.sorted_down:after,
+    .sortable.sorted_up:hover:after {
+        -ms-transform: rotate(180deg);
+        -moz-transform: rotate(180deg);
+        -webkit-transform: rotate(180deg);
+        transform: rotate(180deg);
+    }
+
+    .sortable.sorted_down:hover:after {
+        -ms-transform: rotate(0deg);
+        -moz-transform: rotate(0deg);
+        -webkit-transform: rotate(0deg);
+        transform: rotate(0deg);
+    }
+}
+
+.respond-to-max( @mobile-max, {
+    .table__striped tbody > tr:nth-child(even) {
+        & > th,
+        & > td {
+            background: @table__cell-bg;
+        }
+    }
+    .table__small_stacked {
+        tr,
+        td {
+            display: block;
+        }
+        thead {
+          display: none
+        }
+        td[data-label]:before {
+            .webfont-demi();
+            text-transform: uppercase;
+            margin-top: 0;
+            margin-bottom: .41666667em;
+            font-size: .875em;
+            line-height: 1.83333333;
+            display: block;
+            content: attr(data-label);
+        }
+        td:last-child {
+          margin-bottom: unit( 30px / @base-font-size-px, em );
+        }
+    }
+    .table__small_entry-header {
+        .table__small_stacked();
```

Comment:
Hrm. I understand the justification for not duplicating output, but can duplicated output be justified for ease of use? I'm not saying that extra classes make it that much harder to use, but every extra little "quirk" in the documentation adds up to make these packages pretty complicated.